specification/cosmos-db/resource-manager/Microsoft.DocumentDB/2015-04-08/cosmos-db.json

```diff
@@ -668,6 +668,310 @@
         }
       }
     },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.DocumentDB/databaseAccounts/{accountName}/sourceRegion/{sourceRegion}/targetRegion/{targetRegion}/percentile/metrics": {
```

Comment:
there has to be a segment between percentile and metrics for the percentileName. Is percentile a resource type?

Reply:
may be just call it GET .../targetRegion/{targetRegion}/percentileMetrics instead of percentile/metrics

Reply:
percentile is not a resource type, it is a flag on the metrics handler. The URI is already deployed as /percentile/metrics, so percentileMetrics would not be accurate